test/models/organisation_test.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: organisations
+#
+#  id          :bigint           not null, primary key
+#  name        :string           not null
+#  description :text
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+require 'test_helper'
+
+class OrganisationTest < ActiveSupport::TestCase
+  # test "the truth" do
```

Comment:
@apeksha2697  whenever you get time please add testcases here, or if you need help let me know.
Whenever you'll add testcases, please fix the above fixtures data also